### PR TITLE
feat(jira-issue-found): adding a new output indicating whether the jira issue was found or not

### PR DESCRIPTION
**Pull request by galblond**

### CODE REVIEW

File: src/github-connector.ts [hunk 1]
```diff
@@ -25,31 +25,41 @@ export class GithubConnector {
...
+    let source: ESource | null = null;
 
     switch (WHAT_TO_USE) {
       case ESource.branch:
         keyFound = this.getIssueKeyFromString(branchName);
+        source = ESource.branch;
         break;
       case ESource.prTitle:
         keyFound = this.getIssueKeyFromString(prTitle);
+        source = ESource.prTitle;
```
Comment (cakeinpanic) [inline src/github-connector.ts:44]: hi, to keep consistency with `both` option, you should add here (and above)
```
 source = keyFround ? ESource.prTitle : null;
```

**Review by cakeinpanic — CHANGES_REQUESTED** [review]
thank you for contribution!
please fix a small comment

File: src/github-connector.ts (hunk 1 above)
Comment (galblond) [reply src/github-connector.ts]: Thank you! Done

**Review by cakeinpanic — APPROVED** [review]

File: src/main.ts [hunk 2]
```diff
@@ -35,6 +41,9 @@ async function run(): Promise<void> {
     } else {
       process.exit(0);
     }
+  } finally {
+    core.setOutput('jira-issue-found', jiraIssueFound.toString());
+    core.setOutput('jira-issue-source', jiraIssueSource);
```
Comment (javierjulio) [inline src/main.ts:46]: I believe this change is now causing a deprecation warning:

> Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Comment (cakeinpanic) [reply src/main.ts:46]: you are right, I'd add a fix, thank you

**Pull request merged**